src/ChainSafe.GamingSdk.EVM.Unity/UnityEnvironmentExtensions.cs

```diff
@@ -17,18 +18,18 @@ public static IWeb3ServiceCollection UseUnityEnvironment(this IWeb3ServiceCollec
 
         public static IWeb3ServiceCollection UseUnityEnvironment(this IWeb3ServiceCollection services)
         {
+            services.UseApiAnalytics();
```

Comment:
We should probably document that we enable analytics by default, and that people can opt out by calling `DisableAnalytics`.

Reply:
Should I add an XML comment here or do you mean web documentation?

Reply:
Web documentation. Just something to keep in mind.